kong.conf.default

```diff
@@ -1529,7 +1529,7 @@
                                  # property receives a value (in seconds), it
                                  # will override the TTL for all records.
 
-#dns_stale_ttl = 4               # Defines, in seconds, how long a record will
+#dns_stale_ttl = 86400           # Defines, in seconds, how long a record will
```

Comment:
I have two concerns on this:

1. A large stale TTL may increase the cache memory?
2. For short DNS TTL applications (e.g. `10s`), `24h` may let Kong pick an ancient IP and get `connection refused` error. It may be worth to wait for a while instead of picking an ancient cache.

A value like `60s` more reasonable?

Reply:
I think one day is ok, if the data is expired, we should always use that stale data, so 1 day means alwasy using it. If we set 1 min, there may be not active effect.

For the memory problem, I need to test it.

Reply:
@chobits seems the cache is an LRU cache, so memory might not be an issue.

Reply:
Two small risks:

1. The expired data will be continued to be used if the subsequent DNS resolution fails or there is no active usage of this expired DNS record.
2. The memory usage will be increased with a small amount when worker-level dnscache is filled. The longer stale TTL will make the dnscaceh easier to contain more expired DNS record.
    * If the dnscache is filled, the increased maximum in memory is about `26MB`/worker.

Reply:
The overall memory increase seems not huge.

By the way, what is the diff?
1. the subsequent DNS resolution fails
2. there is no active usage of this expired DNS record.